.github/workflows/ci.yml

```diff
@@ -29,7 +29,9 @@ jobs:
       - name: Build
         run: docker buildx build --build-arg PG_MAJOR=${{ matrix.postgres }} -t test .
       - name: Test
-        run: docker run test cargo pgrx test pg${{ matrix.postgres }}
+        run: |
+          PG_VERSION=$(echo "${{ matrix.postgres }}" | sed 's/[^0-9].*$//')
```

Comment:
Although the image is tagged as `18beta1`, pgrx expects just `18`; this trims off any non-digit from the matrix string.